### PR TITLE
Extract and isolate @afgane's CVMFS feature for galaxyextras

### PR DESCRIPTION
**Pull request by chambm**
* simplify repository list to just data.galaxyproject.org
* add `galaxy_extras_config_cvmfs` option, defaults to false
* add supervisord entry for `autofs`, which CVMFS requires for plug'n'play operation

The autofs stuff should probably be conditional on CVMFS. I wasn't sure whether keeping all 3 stratum1 servers was correct or not, since I removed the test.* and main.* stratum0 servers.

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -219,6 +220,7 @@ supervisor_manage_reports: true
 supervisor_manage_ie_proxy: true
 supervisor_manage_condor: true
 supervisor_manage_cron: true
+supervisor_manage_autofs: true
```
Comment (bgruening) [inline defaults/main.yml]: You need to install automount/autofs isn't it?

**Comment by bgruening** [comment]
I think we fixed the tests. Restarting.

File: defaults/main.yml (hunk 1 above)
Comment (chambm) [inline defaults/main.yml]: I think autofs will come installed by default on Ubuntu 14.04; is the role supposed to be more portable than that?

Comment (bgruening) [inline defaults/main.yml]: Yes. And it is not installed on mine 16.04 ... but maybe this is just my instance.

**Comment by bgruening** [comment]
Here is something wired. Some commits are in this branch which are already in master :(
Can you try to merge master into your branch?

**Comment by chambm** [comment]
I must have screwed up the rebase. If you squash on merge does that matter?

**Comment by bgruening** [comment]
Not sure, I don't want to try this :)

**Comment by bgruening** [comment]
@chambm this one is obsolete?

**Comment by chambm** [comment]
Yes, replaced by #160.

**Pull request closed without merging**